### PR TITLE
Composite Actions Baby Step (Support 1 Run Step)

### PR DESCRIPTION
**Pull request by ethanchewy**
Proof of concept for "Composite Actions".

Not meant for merging into production.

### CODE REVIEW

File: src/Runner.Listener/Runner.cs [hunk 1]
```diff
@@ -37,6 +37,8 @@ public async Task<int> ExecuteCommand(CommandSettings command)
         {
             try
             {
+                _term.WriteLine("Runner Repo TESTING VERSION OF RUNNER...");
+                Trace.Info("Hey I'm in the log file yay!");
```
Comment (ericsciple) [inline src/Runner.Listener/Runner.cs:41]: remove :)

File: src/Runner.Worker/ActionManager.cs [hunk 2]
```diff
@@ -1267,6 +1277,25 @@ public sealed class ScriptActionExecutionData : ActionExecutionData
...
+    {
+        // Do I actually need this??
+        // Pre => pre execution for checking steps before, ibid for post
+        // Look at Script for dummy example
+
+        public override ActionExecutionType ExecutionType => ActionExecutionType.Composite;
+        public override bool HasPre => false;
+        public override bool HasPost => false;
+
+        public List<Pipelines.ActionStep> Steps {get; set;}
```
Comment (ericsciple) [inline src/Runner.Worker/ActionManager.cs:1290]: right cilck -\> format document (will add spaces)

File: src/Runner.Worker/ActionManager.cs [hunk 3]
```diff
@@ -1267,6 +1277,25 @@ public sealed class ScriptActionExecutionData : ActionExecutionData
...
+
+        // public string Script { get; set; }
```
Comment (ericsciple) [inline src/Runner.Worker/ActionManager.cs:1292]: let's remove this commented out code

File: src/Runner.Worker/ActionManager.cs [hunk 4]
```diff
@@ -1267,6 +1277,25 @@ public sealed class ScriptActionExecutionData : ActionExecutionData
         public override bool HasPost => false;
     }
 
+    public sealed class CompositeActionExecutionData : ActionExecutionData
+    {
+        // Do I actually need this??
```
Comment (ericsciple) [inline src/Runner.Worker/ActionManager.cs:1282]: guessing these comments should be deleted now?

**Comment by ethanchewy** [comment]
This is a proof of concept for running single step. Since we've moved on and completed the next goal of supporting multiple run steps, I'm closing this.

**Pull request closed without merging**